ovve_ui/ovve_ui.py

```diff
@@ -497,20 +515,49 @@ def keyPressEvent(self, event):
                 self.showFullScreen()
                 self.fullscreen = True
 
-        if event.key() == QtCore.Qt.Key_Q:
+        elif event.key() == QtCore.Qt.Key_Q:
             self.close()
 
+        elif event.key() == QtCore.Qt.Key_W:
```

Comment:
You wanted to remove these, right?  It could be useful to enable them with a cmdline option for testing